### PR TITLE
chore(docs): update subplat product metrics docs for rps and fxa engineers

### PR DESCRIPTION
**Pull request by biancadanforth**
Closes: #[FXA-8397](https://mozilla-hub.atlassian.net/browse/FXA-8397)

### CODE REVIEW

**Comment by biancadanforth** [comment]
@sean-rose This is ready for you! Unfortunately I couldn't add you as the reviewer in the PR for some reason.

File: docs/relying-parties/tutorials/integration-with-subscription-platform.md [hunk 1]
```diff
@@ -62,6 +64,12 @@ If your integration includes an application service, [subscription state notific
 
 When integrated with the FxA stage or development environments, subscription sign-up's can be tested using [Stripe test cards](https://stripe.com/docs/testing#cards).
 
+#### Ensure your product(s) are included in SubPlat's Looker dashboards
+
+Once you have your products and plans configured in Stripe and/or Contentful, and before you deploy your integration to production, file a [DO ticket in Jira](https://mozilla-hub.atlassian.net/jira/software/c/projects/DO/boards/269) to add your product(s) to the SubPlat ETL.
```
Comment (sean-rose) [inline docs/relying-parties/tutorials/integration-with-subscription-platform.md:69]:
```suggestion
Once you have your products and plans configured in Stripe and/or Contentful, and before you deploy your integration to production, file a [Data Org ticket in Jira](https://mozilla-hub.atlassian.net/jira/software/c/projects/DO/boards/269) to add your product(s) to the SubPlat ETL.
```

**Review by sean-rose — APPROVED** [review]
I appreciate you writing/updating docs for this.

Overall it looks good, just a few minor corrections and suggestions.

File: docs/reference/metrics.md [hunk 2]
```diff
@@ -74,4 +75,66 @@ As noted above, these datasets are restricted but if you have access you'll find
...
+2. In the Explore view in Looker, in the left sidebar, hover over the dimension of interest (e.g. Monthly Active Logical Subscriptions > Subscription > Auto Renew) and click the Info icon, then click "Go to LookML".
+3. You'll be brought to a new page displaying the `view` file. Here you can inspect which table the data came from in the `mozdata` GCP project. Often this can be found by searching for `sql_table_name`. E.g. `mozdata.subscription_platform.monthly_active_logical_subscriptions`.
+4. Now you have one of two choices, though the easiest is likely to search the data catalog ([Acryl](https://mozilla.acryl.io/)) for this dataset by name. The dot notation indicates the namespaces you'll find the table under, with the last value being the table name (e.g. `monthly_active_logical_subscriptions`).
+    - Search for the table name in Acryl.
+    - Filter by Dataset
+    - Click the Lineage tab
+    - Continue to click the `+` icons to the left of the table in question to find upstream tables, until you reach the origin table.
```
[the remark this answers is not in the document]
Comment (biancadanforth) [reply docs/reference/metrics.md:125]: I think there is more than one way to get at this view in the data catalog, but I just re-did it and made the steps much more explicit.

File: docs/relying-parties/how-tos/product-metrics.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+The Subscriptions dashboard leverages one or more Subscription Platform > Logical Subscription(s) explores in Looker. This pulls data indirectly from our subscription providers (i.e. Stripe, Apple and Google), so it allows us to see information like:
+ * Customer and subscription totals and changes over time
+ * Subscription lifecycle events like start, end and plan changes (i.e. upgrades or downgrades)
+
+#### What is a Logical Subscription?
+
+Think of this as a subscription provider's view on a subscription. For example, Stripe doesn't know about bundled subscriptions, so a bundle is considered a single subscription.
+
+Currently, only Stripe and PayPal subscriptions are reflected in these explores. Once [DS-3080](https://mozilla-hub.atlassian.net/browse/DS-3080) and [DS-3081](https://mozilla-hub.atlassian.net/browse/DS-3081) are complete, these explores will also include Google and Apple IAP subscriptions, respectively.
```
[the remark this answers is not in the document]
Comment (biancadanforth) [reply docs/relying-parties/how-tos/product-metrics.md:37]: Thanks! I added this as a caveat to the [SubPlat Subscriptions dashboard](https://mozilla.cloud.looker.com/dashboards/1329) as well in Looker.

**Pull request merged**